segment_sovran-react-native:compileDebugKotlin FAILED  FAILURE: Build failed with an exception

I go this error after running:    **expo run:android**

> Task :segment_sovran-react-native:compileDebugKotlin FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':segment_sovran-react-native:compileDebugKotlin'.

- `analytics-react-native` version: 2.7.0
- Integrations versions (if used):
- React Native version:  0.69.4
- Android
-  Expo version:  "^46.0.0",

--- a/example/package.json
+++ b/example/package.json
@@ -24,7 +24,7 @@
     "@react-native-community/masked-view": "^0.1.11",
     "@react-navigation/native": "^6.0.2",
     "@react-navigation/stack": "^6.0.7",
-    "@segment/sovran-react-native": "0.4.3",
+    "@segment/sovran-react-native": "0.4.4",
     "react": "18.0.0",
     "react-native": "0.69.2",
     "react-native-bootsplash": "^3.2.4",

--- a/packages/core/android/gradle.properties
+++ b/packages/core/android/gradle.properties
@@ -1,4 +1,4 @@
-AnalyticsReactNative_kotlinVersion=1.3.50
+AnalyticsReactNative_kotlinVersion=1.6.0
 AnalyticsReactNative_compileSdkVersion=29
 AnalyticsReactNative_buildToolsVersion=29.0.2
 AnalyticsReactNative_targetSdkVersion=29

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -47,7 +47,7 @@
   "homepage": "https://github.com/segmentio/analytics-react-native#readme",
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.15.17",
-    "@segment/sovran-react-native": "^0.4.3",
+    "@segment/sovran-react-native": "^0.4.4",
     "deepmerge": "^4.2.2",
     "js-base64": "^3.7.2",
     "nanoid": "^3.1.25",

--- a/packages/core/src/info.ts
+++ b/packages/core/src/info.ts
@@ -1,4 +1,4 @@
 export const libraryInfo = {
   name: '@segment/analytics-react-native',
-  version: '2.6.0',
+  version: '2.7.0',
 };

